src/bdev.rs

```diff
@@ -142,23 +161,27 @@ where
     /// If the alias is already present we return true
     pub fn add_alias(&mut self, alias: &str) -> bool {
         let alias = alias.into_cstring();
-        let ret = unsafe { spdk_bdev_alias_add(self.as_ptr(), alias.as_ptr()) }
-            .to_result(Errno::from_i32);
+        let ret =
+            unsafe { spdk_bdev_alias_add(self.as_inner_ptr(), alias.as_ptr()) }
+                .to_result(Errno::from_i32);
```

Comment:
I know it was already like this, but here's a question, what if the alias already exists but it's for another device, do we get a different error other than EEXISTS?

Reply:
No, according to https://github.com/openebs/spdk/blob/v22.01.x-mayastor/lib/bdev/bdev.c#L3294. The only other possible errors are `ENOMEM` or `EINVAL`.